### PR TITLE
Create issue and PR templates

### PR DESCRIPTION
**Pull request by brunobc182**
# What

Create issue and PR templates

# Why

For more concise and reliable information when issues and pull request are created.

# How

- Create issue template file
- Create pull request template file

### CODE REVIEW

**Comment by brunobc182** [comment]
[Criar issue e pull request templates](https://trello.com/c/DhWMga5j/387-criar-issue-e-pull-request-templates)

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Description
+
+_Description of the problem._
+
+# Expected behavior
+
+_Description of what the expected behavior for this._
```
Comment (larimaza) [inline ISSUE_TEMPLATE.md]:
```suggestion
_Description of the expected behavior for this._
```

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Steps to reproduce
+
+_You can describes here step by step how can we reproduce the problem._
```
Comment (larimaza) [inline ISSUE_TEMPLATE.md]:
```suggestion
_Describe step by step how we can reproduce the problem._
```

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Other information
+
+_You can describe here some useful information to help us to resolve this_
```
Comment (larimaza) [inline ISSUE_TEMPLATE.md]:
```suggestion
_You can add useful information here to help us solve this._
```

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+_Description of why you open this pull request._
+
+# How
+
+_You can tell us how you did it._
+
+# Sample
+
+_You can attach some image to show us the result, if applicable._
```
Comment (larimaza) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
_You can attach images to show us the result, if applicable._
```

File: PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# What
+
+_Description of what you did._
+
+# Why
+
+_Description of why you open this pull request._
```
Comment (larimaza) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
_Description of why you're opening this pull request._
```

File: PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+# How
+
+_You can tell us how you did it._
```
Comment (larimaza) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
_Describe your approach to apply these changes._
```

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (brunobc182) [reply ISSUE_TEMPLATE.md]: Done on 75eaf6d511ef7a2f86cefc44230c0c8a975618ed

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (brunobc182) [reply ISSUE_TEMPLATE.md]: Done on 75eaf6d511ef7a2f86cefc44230c0c8a975618ed

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (brunobc182) [reply ISSUE_TEMPLATE.md]: Done on 75eaf6d511ef7a2f86cefc44230c0c8a975618ed

File: PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (brunobc182) [reply PULL_REQUEST_TEMPLATE.md]: Done one a6ec0dc36864062b70aefea00d0d993635ecff02

File: PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (brunobc182) [reply PULL_REQUEST_TEMPLATE.md]: Done one a6ec0dc36864062b70aefea00d0d993635ecff02

File: PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (brunobc182) [reply PULL_REQUEST_TEMPLATE.md]: Done one a6ec0dc36864062b70aefea00d0d993635ecff02

**Pull request merged**